blocks/identity-block/features/login/styles.scss

```diff
@@ -0,0 +1,36 @@
+.xpmedia-subs-login {
```

Comment:
Most, if not all of these styles should be controlled via the `HeadlinedSubmitForm` component

Reply:
ah yep, these are leftover from before i integrated that. I'll remove